Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Start with a Go base image
+FROM golang:1.21
+
+# Install Python, Node.js 16 (LTS version) and Java
+RUN apt-get update && \
+    apt-get install -y gnupg protobuf-compiler python3 python3-pip python3-venv openjdk-17-jdk && \
+    curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
+    apt-get install -y nodejs && \
+    npm install -g npm@8
```

Comment:
Extract specific version

Reply:
There is no specific version. Went with version `11`